web/src/games/mastermind/images.tsx

```diff
@@ -0,0 +1,10 @@
+/* tslint:disable */
+export const IMAGES = {
+    certificate: require('./media/certificate.svg'),
+    circle: require('./media/circle.svg'),
+    heart: require('./media/heart.svg'),
```

Comment:
You can use "import"s instead of "require".

Reply:
I wasn't able to load any SVG, I'll try to use the SeaBattle way and hope it works.